src/Constants.ts

```diff
@@ -140,6 +140,8 @@ const BASE_CONSTANTS: chainConstants = {
   ),
 };
 
+export const TokenIdByChain = (address: string, chainId: number) => `${address.toLowerCase()}-${chainId}`;
```

Comment:
Just noting some places we are checksumming addresses, sometimes they are lowercase, I've been burnt by this before, we'll need to be very careful regarding this